Automattic/wp-calypso
# Edge Cache

#### Proposed Changes

Add edge cache management options. Allows enabling/disabling edge cache and clearing it.

#### Testing Instructions



- Visit Settings -> Performance for any non-Atomic site.
- Try toggling
- `time curl -I https://your-site-goes-here/`
- See `x-ac: MISS` then `x-ac: HIT` when on
- See `x-ac: BYPASS` when off
- Clear cache
- See `x-ac: MISS` then `x-ac: HIT` when on
